lnrpc/walletrpc/walletkit.proto

```diff
@@ -397,13 +404,54 @@ message Account {
     // Whether the wallet stores private keys for the account.
     bool watch_only = 8;
 }
+
+message AddressProperty {
+    // The address encoded using a bech32 format.
```

Comment:
This isn't true for all address types. Could change this to `The address encoded using the appropriate format depending on the address type (base58, bech32, bech32m).`

Reply:
Sure!